src/plugins/data_table/data_table.js

```diff
@@ -271,6 +276,10 @@ var LocalDataTable = (function() {
       this._columnManager.on("change:visibility", this._onColumnVisibilityChange);
       this._columnManager.applyVisibilityPreferences();
       if (this.collection.length) this._onReset(this.collection);
+      // if resizeable, add resizeable class
+      if (this._colReorder && this._colReorder.s.allowResize) {
+        this.$("table").addClass('dataTable-resizeableColumns')
```

Comment:
weird that hound didn't comment on this line (or maybe it did and it's been deleted). But please keep the quoting consistent for now with double quotes